apps/vs-code-react/src/app/export/status/status.tsx

```diff
@@ -26,28 +39,34 @@ export const Status: React.FC = () => {
         {intlText.EXPORT_STATUS_TITLE}
       </Text>
       <List items={statuses} onRenderCell={renderStatus} />
-      <FinalStatusGadget />
+      <FinalStatusGadget finalStatus={finalStatus} />
     </div>
   );
 };
 
-const FinalStatusGadget: React.FC = () => {
+interface FinalStatusGadgetProps {
+  finalStatus: string | undefined;
+}
+
+const FinalStatusGadget: React.FC<FinalStatusGadgetProps> = ({ finalStatus }) => {
   const intl = useIntl();
   const workflowState = useSelector((state: RootState) => state.workflow);
-  const status = workflowState.finalStatus;
   const { targetDirectory } = workflowState.exportData;
 
   const message = intl.formatMessage({
-    defaultMessage: 'The selected workflows exported successfully. For next steps, review the ',
+    defaultMessage: 'For next steps, review the ',
     description: 'The success message.',
   });
 
-  switch (status) {
+  switch (finalStatus) {
     case FinalStatus.Succeeded:
       return (
-        <Text block>
-          {message} {targetDirectory.path}/.logs/export/README.md
-        </Text>
+        <div className="msla-export-status--item">
+          <Icon iconName={'SkypeCheck'} />
+          <Text style={{ marginLeft: '5px' }}>
+            {message} {targetDirectory.path}/.logs/export/README.md
```

Comment:
We should have this path "/.logs/export/README.md" as a constant so others touching this code have an idea of what it really means/does

Reply:
Sure! thanks